endpoints-support.md

```diff
@@ -50,29 +50,29 @@ OneviewSDK::Datacenter.find_by(@client, width: 11000).map(&:remove)
-|<sub>/rest/enclosure-groups/{id}</sub>                                                   | PUT         | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
-|<sub>/rest/enclosure-groups/{id}</sub>                                                   | DELETE      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
-|<sub>/rest/enclosure-groups/{id}/script</sub>                                            | GET         | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
-|<sub>/rest/enclosure-groups/{id}/script</sub>                                            | PUT         | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
+|<sub>/rest/enclosure-groups</sub>                                                        | GET         | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
+|<sub>/rest/enclosure-groups</sub>                                                        | POST        | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
+|<sub>/rest/enclosure-groups/{id}</sub>                                                   | GET         | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
+|<sub>/rest/enclosure-groups/{id}</sub>                                                   | PUT         | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
+|<sub>/rest/enclosure-groups/{id}</sub>                                                   | DELETE      | :white_check_mark:   | :white_check_mark:   | :white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |:white_check_mark:   |
+|<sub>/rest/enclosure-groups/{id}/script</sub>                                            | GET         | :white_check_mark:   | :white_check_mark:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   | :heavy_minus_sign:   |
```

Comment:
Please revert this as these endpoints are present in C7000

Reply:
Done